lib/config/index.js

```diff
@@ -36,6 +38,20 @@ function parseConfigs(env, argv) {
   // Get global config
   logger.debug(`raw config=${redact(config)}`);
 
+  if (config.autodiscover) {
+    if (config.platform === 'github') {
+      logger.info('Autodiscovering GitHub repositories');
+      config.repositories = await githubApi.getRepos(config.token, config.endpoint);
+    } else if (config.platform === 'gitlab') {
+      logger.info('Autodiscovering GitLab repositories');
+      config.repositories = await gitlabApi.getRepos(config.token, config.endpoint);
+    } else {
+      logger.error(`Unknown platform ${config.platform}`);
```

Comment:
`Can't use 'autodiscover' feature as you're using an unsupported platform for this feature`